### ISSUE TITLE
Tshark on Linux

### ISSUE THREAD

**Issue by OrlandoST** [telekom/5g-trace-visualizer#18]
This code dont working jn Linux, because there isnt Wireshark Portable for Linux and OS path thshark is incorrect

I fix this problem by adding follow code in call_wireshark_for_one_version():
```python
    # Add option to not use a Wireshark portable version but rather the OS-installed one
    if wireshark_version == 'OS':
        if (platform == 'Linux'):
            tshark_path = "/usr/bin/tshark"
        else:
            tshark_path = os.path.join('tshark')
    else:
        tshark_path = os.path.join(get_wireshark_portable_folder(wireshark_version), 'tshark')
    logging.debug('tshark path: {0}'.format(tshark_path))
```

I can create pull request for this

**Comment by jkolom** [comment]
Hi Pavel,
The fix suggested by a previous user (see #4) only worked if ``tshark`` is in the path. There was not much thought of Linux when I put it together, I have to admit. If the fix helps, happy to get a pull request :)

**Comment by OrlandoST** [comment]
Fixed in 42224335b3f9a52123381178fe1638e910a27d57
